components/docs.css

```diff
@@ -0,0 +1,47 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+html {
+    font-size: 16px;
+}
+
+.side-bar {
+    display: flex;
+    flex-direction: column;
+    margin-top: 8rem;
+    gap: 2rem;
+    margin-left: 1.5rem;
+    margin-right: 1.5rem;
```

Comment:
You could use the margin 3 valued shorthand property here. 
`margin: 8rem 1.5rem initial`
i.e.
`margin: [margin-top]    [margin-left-right]    [margin-bottom]`